api/messages.proto

```diff
@@ -47,10 +66,15 @@ message Assignment {
   string connection = 1;
```

Comment:
do we care about explicitly having a connection / error any more? or just have an any that can have whatever? since we are changing this, lets revisit / finalize this - @calebatwd ?

Reply:
For this change, keeping this around.